### PR TITLE
Remove ADClientMetrics

### PR DESCRIPTION
**Pull request by antrix1989**
We don't use these headers for telemetry. Also, there is the issue with sending localized headers to the server.

### CODE REVIEW

**Comment by oldalton** [comment]
@antrix1989, please don't merge this directly to master.
Follow the hotfix git flow:
1. Create a hotfix branch from master (e.g. hotfix/<version>)
2. Create a new PR with your changes against hotfix branch
3. Merge your PR
4. Update version numbers and changelog in the hotfix branch
5. Create a PR to merge hotfix into master
6. Merge hotfix into master. Tag the release and publish to CocoaPods.

File: ADAL/ADAL.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1099,8 +1091,6 @@
 		B258488020747A54007FAD22 /* KeyVault.framework */ = {isa = PBXFileReference; explicitFileType = wrapper.framework; path = KeyVault.framework; sourceTree = BUILT_PRODUCTS_DIR; };
 		B258488220747B01007FAD22 /* KeyVaultClient.xcodeproj */ = {isa = PBXFileReference; lastKnownFileType = "wrapper.pb-project"; name = KeyVaultClient.xcodeproj; path = ../KeyVaultClient/KeyVaultClient.xcodeproj; sourceTree = "<group>"; };
 		B26207E022C872DA00F867D9 /* ADEnrollmentGateway+UnitTests.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = "ADEnrollmentGateway+UnitTests.h"; sourceTree = "<group>"; };
-		B267CA191EE0E9FF00C0B5A8 /* ADNegotiateHandler.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = ADNegotiateHandler.h; sourceTree = "<group>"; };
```
Comment (oldalton) [inline ADAL/ADAL.xcodeproj/project.pbxproj:1102]: why did you remove this one?

Comment (antrix1989) [reply ADAL/ADAL.xcodeproj/project.pbxproj:1102]: I didn't. Seems it was removed a while ago, but it was still in the proj file by some reason.

File: ADAL/resources/mac/Info.plist [hunk 2]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>4.0.6</string>
+	<string>4.0.8</string>
```
Comment (oldalton) [inline ADAL/resources/mac/Info.plist:18]: should it be 4.0.9?
Also, can you update version numbers in other places as well :)?
